File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/core/PropertiesModifier.java
package app.zxtune.core;

/**
 * Properties modifier interface
 */
public interface PropertiesModifier {

  /**
   * Setting integer property
   *
   * @param name Name of the property
   * @param value Value of the property
   */
  void setProperty(String name, long value);

  /**
   * Setting string property
   *
   * @param name Name of the property
   * @param value Value of the property
   */
  void setProperty(String name, String value);
}

File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/core/PropertiesModifier.kt
package app.zxtune.core

/**
 * Properties modifier interface
 */
interface PropertiesModifier {
    /**
     * Setting integer property
     *
     * @param name Name of the property
     * @param value Value of the property
     */
    fun setProperty(name: String, value: Long)

    /**
     * Setting string property
     *
     * @param name Name of the property
     * @param value Value of the property
     */
    fun setProperty(name: String, value: String)
}
